packages/bun-usockets/src/crypto/root_certs.cpp

```diff
@@ -5,8 +5,12 @@
 #include "./internal/internal.h"
 #include <atomic>
 #include <string.h>
-static const int root_certs_size = sizeof(root_certs) / sizeof(root_certs[0]);
+#ifdef __APPLE__
+#include <Security/Security.h>
```

Comment:
We don't currently link Security.framework into Bun's executable and it would cost a little bit on startup. Can we instead do this with dlopen?